wordpress-mobile/WordPress-iOS
# Initial changes to make the Today Home Widget customizable.

First attempt at making our Today Home Widget customizable.

In this PR you can:

- Customize the widget to select which site to show the stats for.
- See the site stats in the widget.

This PR also adds a new target (the Intents Extension) together with provisioning profiles and App ID.

## About the number of lines changed:

The PR is quite large, with around 2000 lines added.  Most of these, though, are changes to add the new intents target and the certificates.  The code changes aren't that many.

## Known limitations:

- This PR doesn't change the stats refresh logic so the info may not be up to date at all times.
- Remote icons for sites aren't loading yet: Apple's default mechanism for downloading remote images in the intents extension is not working at all, and I'm a bit stuck trying to figure out the best solution.
- It seems we're showing hidden sites too.  We should probably filter the list of sites shown to visible ones.

## To test:

1. Long tap on the widget.
2. Tap "Edit Widget"
3. Tap "Choose" in the "Site" row.
4. Select a site from your list.

Make sure the site you selected is properly loaded in the widget.

## PR submission checklist:
